pinot-integration-tests/src/test/integration-test-suite-1.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="integration-suite-1">
+  <test name="integration-suite-1">
+    <groups>
+      <run>
+        <include name="integration-suite-1"/>
+      </run>
+    </groups>
+    <packages>
+      <package name="org.apache.pinot.integration.tests.*"/>
```

Comment:
this looks like in includes everything (*) and the other `*-2` also have the same package name, do we plan to still do the first letter split?

Reply:
nvm i actually see you do exclusion from the `*-2` xml... (maybe to call it out in comment)